src/endpoints/auth/auth.controller.ts

```diff
@@ -8,8 +8,7 @@ import { EmailEnum, OpenAPITagsEnum } from '../../shared/enums';
 import { createError } from '../../shared/errors';
 import { statusOutputSchema } from '../../shared/models/outputs';
 import { EmailService, HashingService, JWTService, LocalStorage, VerificationCodeService } from '../../shared/services';
-import { LoggerInstance } from '../../shared/services/logger';
-import { IBodyForgotEmail, IBodySignUp, IBodyValidateEmail, IHeadersValidateEmail } from './inputs';
+import { IBodyCodeEmail, IBodyForgotEmail, IBodySignUp, IBodyValidateEmail, IHeadersValidateEmail } from './inputs';
```

Comment:
Maybe this level of abstraction is too much? Lots of these interfaces are objects with just 1 field